apps/dotcom/client/src/tla/components/TlaEditor/TlaSnapshotsEditor.tsx

```diff
@@ -0,0 +1,83 @@
+import { getLicenseKey } from '@tldraw/dotcom-shared'
```

Comment:
nit: can we rename this file to `TlaPublishedEditor`? throwing me off, heh (especially because it's explicitly read-only, it can kiiinda sound like it's editable but it's not)